### DISCUSSION TITLE
重复：无法复现的字体问题

### DISCUSSION THREAD

**Discussion by linger040831** [hooke007/mpv_PlayKit#245] [回收]
![屏幕截图_20221214_122049](https://user-images.githubusercontent.com/84896599/207506270-f525601a-54f3-41ed-bd02-779bbbddb240.png)
![屏幕截图_20221214_122057](https://user-images.githubusercontent.com/84896599/207506280-a544d213-d556-4fa5-bf10-7e3321427477.png)
![屏幕截图_20221214_122123](https://user-images.githubusercontent.com/84896599/207506285-6029a467-65b6-4ccd-b1f9-f48bdf57f18e.png)
![image](https://user-images.githubusercontent.com/84896599/207506523-27005551-3357-4d32-8058-866ebfccb628.png)

这是[mpv-lazy-2022EX](https://github.com/hooke007/MPV_lazy/releases/tag/20221112)这个包测试的，osc的字体是正常的，但是名字是LG Amaze Dolby Vision UHD 4K Demo 顶栏字体就是不正常的，改成中文是没问题的

![image](https://user-images.githubusercontent.com/84896599/207507050-7e8925ec-b80f-4e7b-be45-f024cc528ca8.png)

原版osc用LG Amaze Dolby Vision UHD 4K Demo这个名字就没什么问题

**Comment by linger040831** [comment]
好像英文的字体都不会改变？右键菜单的英文字体也不正常。
![image](https://user-images.githubusercontent.com/84896599/207507622-97720e47-251e-45a1-8173-950ec04846ae.png)

**Comment by hooke007** [comment]
我依然无法复现你的问题
![Snipaste_2022-12-14_04-54-37](https://user-images.githubusercontent.com/41094733/207509243-85e68c1d-29f6-45f4-b6f1-3e303df1afe1.jpg)

**Reply by linger040831** [reply]
我也不知道为什么会这样，我用新下载的包什么参数都没改😂

**Reply by hooke007** [reply]
另外这种重复帖不要新开档，直接在版本反馈下发
